### DISCUSSION TITLE
[Feature Request] Support Send api

### DISCUSSION THREAD

**Discussion by markcs** [dani-garcia/vaultwarden#1483] [Ideas]
Bitwarden have introduced Send.  Refer to https://bitwarden.com/products/send/

When trying this functionality from Android, I see the following error in the server logs

```
[2021-03-12 23:37:08.973][request][INFO] POST /api/sends
[2021-03-12 23:37:08.973][response][INFO] 404 Not Found
```

It would be great if it could be implemented in bitwarden_rs

**Comment by char** [comment]
The lead developer has already posted a stance on this:

https://github.com/dani-garcia/bitwarden_rs/issues/246#issuecomment-796966385

> I'll take a stab at implementing it over the weekend.

**Comment by BlackDex** [comment]
This has been mentioned a few times already. https://github.com/dani-garcia/bitwarden_rs/issues/246#issuecomment-796966385

**Reply by knorrre** [reply]
Unfortunately I am a bit inexperienced regarding programming. Nevertheless, I managed to run bitwarden_rs on a Synology NAS using Docker.
Now I saw in the wiki that the new feature "Send" was also implemented in bitwarden_rs. However, the current version is still 1.19.0. How can I update the container to use Send?

**Reply by enizor** [reply]
See #1498:
The Send feature is too recent and is only available in the `bitwardenrs/server:testing` container (unstable)
